scripts/buildsystems/vcpkg.cmake

```
@@ -487,40 +487,59 @@ if(VCPKG_MANIFEST_MODE AND VCPKG_MANIFEST_INSTALL AND NOT Z_VCPKG_CMAKE_IN_TRY_C
             set(Z_VCPKG_MANIFEST_INSTALL_ECHO_PARAMS)
         endif()
 
-        execute_process(
-            COMMAND "${Z_VCPKG_EXECUTABLE}" install
-                --triplet "${VCPKG_TARGET_TRIPLET}"
-                --vcpkg-root "${Z_VCPKG_ROOT_DIR}"
-                "--x-wait-for-lock"
-                "--x-manifest-root=${VCPKG_MANIFEST_DIR}"
-                "--x-install-root=${_VCPKG_INSTALLED_DIR}"
-                "${Z_VCPKG_FEATURE_FLAGS}"
-                ${Z_VCPKG_ADDITIONAL_MANIFEST_PARAMS}
-                ${VCPKG_INSTALL_OPTIONS}
-            OUTPUT_VARIABLE Z_VCPKG_MANIFEST_INSTALL_LOGTEXT
-            ERROR_VARIABLE Z_VCPKG_MANIFEST_INSTALL_LOGTEXT
-            RESULT_VARIABLE Z_VCPKG_MANIFEST_INSTALL_RESULT
-            ${Z_VCPKG_MANIFEST_INSTALL_ECHO_PARAMS}
-        )
+        file(TIMESTAMP "${VCPKG_MANIFEST_DIR}/vcpkg.json" Z_VCPKG_MANIFEST_TIMESTAMP UTC)
+        if(EXISTS "${VCPKG_MANIFEST_DIR}/vcpkg-configuration.json")
+            file(TIMESTAMP "${VCPKG_MANIFEST_DIR}/vcpkg-configuration.json" Z_VCPKG_CONFIGURATION_TIMESTAMP UTC)
+        else()
+            set(Z_VCPKG_CONFIGURATION_TIMESTAMP "NOTFOUND")
+        endif()
 
-        file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/vcpkg-manifest-install.log" Z_VCPKG_MANIFEST_INSTALL_LOGFILE)
-        file(WRITE "${Z_VCPKG_MANIFEST_INSTALL_LOGFILE}" "${Z_VCPKG_MANIFEST_INSTALL_LOGTEXT}")
+        set(Z_VCPKG_DO_ACTUAL_INSTALL OFF)
+        foreach(var IN ITEMS ADDITIONAL_MANIFEST_PARAMS FEATURE_FLAGS TARGET_TRIPLET MANIFEST_TIMESTAMP CONFIGURATION_TIMESTAMP)
```

```suggestion
        foreach(var IN ITEMS ADDITIONAL_MANIFEST_PARAMS FEATURE_FLAGS TARGET_TRIPLET Z_VCPKG_MANIFEST_TIMESTAMP Z_VCPKG_CONFIGURATION_TIMESTAMP)
```

I think this is what you meant?

---

no, each of these is `Z_VCPKG_${var}`